app/views/layouts/workspace_channel.html.erb

```diff
@@ -6,6 +6,7 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;500;600;700&display=swap" rel="stylesheet">
```

Comment:
is this needed? I thought you were importing the fonts from the CSS file you had created

Reply:
no no I deleted from the CSS, the target is to remove that CSS file